rpc_jobs/defaults.yml

```diff
@@ -21,3 +21,7 @@
     JENKINS_NODE_EXECUTORS: "2"
     jenkins_node_exclusive: true
     allow_jenkins_sudo: true
+    # rpc_leapfrog_params
+    LEAPFROG_TEST_NETWORKS: "0"
```

Comment:
Same remark here for the L25-27.

I'd rename rpc_leapfrog_params L24 to load_generator_params